bin/BuildPackages.sh

```diff
@@ -184,10 +210,36 @@ do
         *)
           run_configure_and_make
         ;;
-      esac;
+      esac
       ) || build_fail
+}
+
+      (buildpackage > >(tee "$LOGDIR/$dir.out") 2> >(tee "$LOGDIR/$dir.err" >&2) )> >(tee "$LOGDIR/$dir.log" ) 2>&1
       # end subshell
+      # remove superfluous log files if there was no error message
+      if [[ ! -s "$LOGDIR/$dir.err" ]]
+      then
+        rm -f "$LOGDIR/$dir.err"
+        rm -f "$LOGDIR/$dir.out"
+      fi
     else
       echo "$dir is not a GAP package -- no PackageInfo.g"
```

Comment:
Well, it seemed like the sensible thing to do at the time, but I'm equally happy to silently skip directories with no `PackageInfo.g`, particularly if there is a good reason (and the existence of `.git` / `.hg` directories seems like a good reason).